applications/RomApplication/python_scripts/fluid_dynamics_analysis_rom.py

```diff
@@ -0,0 +1,44 @@
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication
+import KratosMultiphysics.RomApplication as romapp
+from KratosMultiphysics.RomApplication import python_solvers_wrapper_rom as solver_wrapper
+from KratosMultiphysics.FluidDynamicsApplication.fluid_dynamics_analysis import FluidDynamicsAnalysis
+
+import json
+
+class FluidDynamicsAnalysisROM(FluidDynamicsAnalysis):
+
+    def __init__(self,model,project_parameters):
+        super().__init__(model,project_parameters)
+
+    #### Internal functions ####
+    def _CreateSolver(self):
+        """ Create the Solver (and create and import the ModelPart if it is not alread in the model) """
+        ## Solver construction
+        with open('RomParameters.json') as rom_parameters:
+            rom_settings = KratosMultiphysics.Parameters(rom_parameters.read())
+            self.project_parameters["solver_settings"].AddValue("rom_settings", rom_settings["rom_settings"])
+        return solver_wrapper.CreateSolverByParameters(self.model, self.project_parameters["solver_settings"],self.project_parameters["problem_data"]["parallel_type"].GetString())
+
+    def _GetSimulationName(self):
+        return "::[ROM Simulation]:: "
+
+    def ModifyInitialGeometry(self):
+        """Here is the place where the BASIS_ROM and the AUX_ID are imposed to each node"""
+        super().ModifyInitialGeometry()
+        computing_model_part = self._solver.GetComputingModelPart()
+        with open('RomParameters.json') as f:
+            data = json.load(f)
+            nodal_dofs = len(data["rom_settings"]["nodal_unknowns"])
+            nodal_modes = data["nodal_modes"]
+            counter = 0
+            rom_dofs= self.project_parameters["solver_settings"]["rom_settings"]["number_of_rom_dofs"].GetInt()
+            for node in computing_model_part.Nodes:
+                aux = KratosMultiphysics.Matrix(nodal_dofs, rom_dofs)
+                for j in range(nodal_dofs):
+                    Counter=str(node.Id)
```

Comment:
Why casting to string?

Reply:
We are calling the key in the JSON file in which the nodal modes are contained, that's why it is a string...